WordPress/src/main/java/org/wordpress/android/viewmodel/posts/PostListItemUiStateHelper.kt

```diff
@@ -98,7 +98,10 @@ class PostListItemUiStateHelper @Inject constructor(private val appPrefsWrapper:
         )
         val statusesDelimeter = UiStringRes(R.string.multiple_status_label_delimiter)
         val onSelected = {
-            onAction.invoke(post, BUTTON_EDIT, POST_LIST_ITEM_SELECTED)
+            when (postStatus) {
```

Comment:
I'd consider listing all the enum values instead of using `else` case. We use lint rule which prevents using "default" case in `switch` statements, however, we don't have the same rule for `when` statements. :(
I think it's a good practice as when a new value is added to the enum the author is forced to take into consideration all the `when`/`switch` statements and implement it correctly. Wdyt?

Reply:
updated in 480e337